src/CMakeLists.txt

```diff
@@ -153,14 +153,18 @@ else ()
 endif (PGINDENT)
 
 # Configuration for running pgindent
-if (OBJDUMP AND PGINDENT)
-  add_custom_command(OUTPUT typedefs.list
-    DEPENDS ${PROJECT_NAME}
-    COMMAND sh ${CMAKE_BINARY_DIR}/scripts/generate_typedefs.sh > typedefs.list)
-  add_custom_target(pgindent
-    COMMAND ${PGINDENT} -typedefs typedefs.list -code-base ${CMAKE_SOURCE_DIR}/src
-    DEPENDS typedefs.list)
-endif (OBJDUMP AND PGINDENT)
+if (PGINDENT)
+  if (OBJDUMP)
+    add_custom_command(OUTPUT typedefs.list
+      DEPENDS ${PROJECT_NAME}
+      COMMAND sh ${CMAKE_BINARY_DIR}/scripts/generate_typedefs.sh > typedefs.list)
+    add_custom_target(pgindent
+      COMMAND ${PGINDENT} -typedefs typedefs.list -code-base ${CMAKE_SOURCE_DIR}/src
+      DEPENDS typedefs.list)
+  else()
+	message(STATUS "Cannot use pgindent -- found pgindent, but objdump not installed.")
```

Comment:
So, we already have an error if `OBJDUMP` is not set in `scripts/CMakeLists.txt`. Not sure how this one improves on that?

I think what we may want to do here is to install a `pgindent` make target also in the negative case, which outputs an informative message when you do `make pgindent`. I think such a runtime error is more helpful.

Reply:
Oh wow, I completely missed the `scripts/CMakeLists.txt` message. You're right, this means this compile-time error will also be missed. Added runtime error instead.